sys/conf/options.arm64

```diff
@@ -1,6 +1,7 @@
 # $FreeBSD$
 
 ARM64				opt_global.h
+
```

Comment:
This seems unneeded?

Reply:
Looks like it was missed when I removed an old `MORELLO` option.